modules/test/playwright/tests/frontend-editor-ckeditor-web/react.spec.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {ckeditorSamplePageTest} from './fixtures/ckeditorSamplePageTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	isolatedSiteTest,
+	loginTest(),
+	ckeditorSamplePageTest
+);
+
+test.beforeEach(async ({ckeditorSamplePage, page, site}) => {
+	await ckeditorSamplePage.createAndGotoSitePage({site});
```

Comment:
Can we reuse this in [classic.spec.ts](https://github.com/liferay/liferay-portal/blob/1f96376efb0f1977b607bd98cf55c4b075245619/modules/test/playwright/tests/frontend-editor-ckeditor-web/classic.spec.ts#L33-L43) and [lecagcy.spec.ts](https://github.com/liferay/liferay-portal/blob/1f96376efb0f1977b607bd98cf55c4b075245619/modules/test/playwright/tests/frontend-editor-ckeditor-web/legacy.spec.ts#L35-L45)? They look very similar

Reply:
Yep! We can do this in a followup, I figured PR is already big enough.